src/main/java/org/opentripplanner/hasura_client/VehiclePositionsGetter.java

```diff
@@ -0,0 +1,66 @@
+package org.opentripplanner.hasura_client;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import org.opentripplanner.hasura_client.hasura_objects.Vehicle;
+import org.opentripplanner.hasura_client.mappers.HasuraToOTPMapper;
+import org.opentripplanner.hasura_client.mappers.VehiclePositionsMapper;
+import org.opentripplanner.routing.core.vehicle_sharing.VehicleDescription;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class VehiclePositionsGetter extends HasuraGetter<VehicleDescription, Vehicle> {
+    private static final Logger LOG = LoggerFactory.getLogger(VehiclePositionsGetter.class);
+
+    @Override
+    protected String query() {
+        return
+                "{\"query\": \"query VehiclesForArea($latMin: float8, $lonMin: float8, $latMax: float8, $lonMax: float8) {\\n" +
+                        "  items:vehicles(\\n" +
+                        "    where: {\\n" +
+                        "    latitude: {\\n" +
+                        "      _gte: $latMin\\n" +
+                        "      _lte: $latMax\\n" +
+                        "    }\\n" +
+                        "    longitude: {\\n" +
+                        "      _gte: $lonMin\\n" +
+                        "      _lte: $lonMax\\n" +
+                        "    }\\n" +
+                        "    provider:{\\n" +
```

Comment:
To jest zastępowanie jednego rozwiązania danego feature'a drugim i to w dodatku bez mówienia o tym komukolwiek xD. Ja jestem jak najbardziej za, ale jak tak robisz, to usuń ściąganie tego pola `available` (kilka linijek niżej), jego obecność w DTO oraz użycie w mapperze i testy. Najlepiej w oddzielnym PRze

Reply:
Przeniosłem te zmiany na osobnego brancha i dokończyłem. Zamierzałem wam napisać o tej zmianie przed wystawieniem ale zapomniałem :|